src/Data/Fin/Properties.agda

```diff
@@ -674,6 +682,12 @@ combine-surjective {m} {n} i with remQuot {m} n i | P.inspect (remQuot {m} n) i
   (uncurry remQuot-combine)
   (combine-remQuot {m} n)
 
+Fin[m^n]↔Fin[m]^n : ∀ m n → Fin (m ^ n) ↔ Fin m Vec.^ n
```

Comment:
Sorry, one last comment. I've just noticed that `Data.Vec.Recursive` depends on `Data.Fin.Base`. Therefore `Data.Vec.Properties` should also depend on `Data.Fin.Properties`. However, this PR introduces a dependency the other way around which is going to cause problems in future. To fix this the proof needs moving to `Data.Vec.Recursive`. (don't forget to update the CHANGELOG as well).